src/components/Editor.vue

```diff
@@ -51,8 +52,21 @@
 							:has-connection-issue="hasConnectionIssue" />
 					</ReadonlyBar>
 				</div>
+				<!-- Document is not read-only, but open as read-only is enabled -->
+				<div v-else-if="!documentReadOnly && openReadOnlyEnabled && !editable" class="text-editor--editable-bar-edit">
+					<EditableBarEdit>
+						<Status :document="document"
+							:dirty="dirty"
+							:sessions="filteredSessions"
+							:sync-error="syncError"
+							:has-connection-issue="hasConnectionIssue" />
+					</EditableBarEdit>
+				</div>
 				<!-- Rich Menu -->
 				<template v-else>
+					<div v-if="openReadOnlyEnabled" class="text-editor--editable-bar-save">
+						<EditableBarSave />
```

Comment:
UI wise I would prefer to not have an extra menu line but rather integrate this into the existing menu bar This could be passed into a slot of the Menubar component